### ISSUE TITLE
Readme needs updating

### ISSUE THREAD

**Issue by ShantanuNair** [Lord-Kamina/Deluge-Magnet-Handler#6]
Readme needs updating w respect to instructions. .app is no longer in the bundle.

**Comment by jrtnq514** [comment]
Download one of the release zips.
https://github.com/Lord-Kamina/Deluge-Magnet-Handler/releases

**Comment by Lord-Kamina** [comment]
@jrtnq514 Indeed. Thanks for answering this; I'd seen this and could have sworn I had replied. D:
